bootstrap/pkg/utils/k8utils.go

```diff
@@ -325,3 +145,198 @@ func DeleteResourceFromFile(config *rest.Config, filename string) error {
 	}
 	return nil
 }
+
+type Apply struct {
```

Comment:
Just a thought. Found it very confusing to have name "Apply".Can this be renamed to Applykubectl or KubectlApply? There are too many "apply"s now in kfctl so was thinking that renaming will help.

Reply:
How about ApplyResources, similar to CreateResources and DeleteResources that we have now?